internal/otelutil/baggage.go

```diff
@@ -0,0 +1,44 @@
+package otelutil
+
+import (
+	"context"
+
+	"github.com/containerd/log"
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel/baggage"
+)
+
+// TriggerKey is the key used for the 'trigger' member in the baggage. It is
+// used to know what triggered a code path (e.g. API call, libnet init, etc...)
+const TriggerKey = "trigger"
+
+// MustNewBaggage creates an OTel Baggage containing the provided members. It
+// panics if the baggage cannot be created.
+//
+// DO NOT USE this function with dynamic values.
+func MustNewBaggage(members ...baggage.Member) baggage.Baggage {
+	b, err := baggage.New(members...)
+	if err != nil {
+		log.G(context.Background()).WithFields(logrus.Fields{
+			logrus.ErrorKey: err,
+			"members":       members,
+		}).Fatal("OTel baggage creation failure")
```

Comment:
Trying to avoid direct imports of logrus; could you use `log.Fields` here? And for `logrus.ErrorKey`, we can just use `"error"` (we know the `ErrorKey` won't change, so it's ok to hard-code it to `"error"`);

```suggestion
		log.G(context.Background()).WithFields(log.Fields{
			"error":   err,
			"members": members,
		}).Fatal("OTel baggage creation failure")
```

Reply:
Done